tools/clone_nonbillables_and_process.sh

```diff
@@ -28,9 +28,19 @@ if [ ! -d ./non-billable-projects ]; then
 fi
 
 INVOICE_MONTH=$(date --date="$(date +%Y-%m-01) -1 month" +%Y-%m)
+export S3_ENDPOINT
```

Comment:
These exports are unnecessary as these valued will not be assigned in the script but outside of it.